ee-9/feature-pack/pom.xml

```diff
@@ -576,7 +576,7 @@
             <exclusions>
                 <exclusion>
                     <groupId>org.wildfly.core</groupId>
-                    <artifactId>wildfly-elytron-integration</artifactId>
+                    <artifactId>wildfly-elytron-integration-legacy-namespace</artifactId>
```

Comment:
This can be removed altogether. The https://github.com/wildfly/wildfly-core/blob/main/core-feature-pack/galleon-common/pom.xml doesn't declare this dependency.

Reply:
Removed, thanks.